tests/js/spec/components/featureTourModal.spec.jsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+
+import {mountWithTheme} from 'sentry-test/enzyme';
+
+import FeatureTourModal from 'app/components/modals/featureTourModal';
+import GlobalModal from 'app/components/globalModal';
+
+const steps = [
+  {
+    title: 'First',
+    body: 'First step',
+    image: <em data-test-id="step-image">Image</em>,
+    actions: (
+      <a href="#" data-test-id="step-action">
+        additional action
+      </a>
+    ),
+  },
+  {title: 'Second', body: 'Second step'},
+];
+
+describe('FeatureTourModal', function() {
+  let onAdvance, onCloseModal;
+
+  const createWrapper = () =>
+    mountWithTheme(
+      <React.Fragment>
+        <GlobalModal />
+        <FeatureTourModal steps={steps} onAdvance={onAdvance} onCloseModal={onCloseModal}>
+          {({handleShow}) => (
+            <a href="#" onClick={handleShow} data-test-id="reveal">
+              Open
+            </a>
+          )}
+        </FeatureTourModal>
+      </React.Fragment>
+    );
+
+  const showModal = async wrapper => {
+    wrapper.find('[data-test-id="reveal"]').simulate('click');
+    await tick();
+    await wrapper.update();
+  };
+
+  beforeEach(function() {
+    onAdvance = jest.fn();
```

Comment:
as they are mock functions and do not change ... would it be better to just set them at the top and change let to const?